applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1230,34 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+=====================
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+Complete the following steps to test upgrading the firmware on both cores at the same time on a gateway device (the procedure is identical for upgrading headset firmware):
+
+1. Install `nRF Connect Device Manager`_ on your Android or iOS device.
+#. Connect an external flash shield to the gateway. See :ref:`nrf53_audio_app_configuration_configure_fota` for details.
```

Comment:
External flash is also a requirement that should go into the "Requirements for FOTA" section.
The configuration link should be mentioned as step 1: "Make sure you <configured the gateway for DFU using external flash memory>."
I assume that "make sure the gateway runs a fw that supports external flash memory" means configuring the Kconfigs mentioned here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf5340.html#ug-nrf5340-multi-image-dfu If so, we need this mentioned explicitly in the Configuring FOTA section.

Reply:
Added external flash shield to the "Requirements for FOTA" section.
Made the configuration link as step 1.
See next comment